### DISCUSSION TITLE
Welcome to BATE-PONTO-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by PedroMiguel7** [PedroMiguel7/BATE-PONTO-Frontend#42] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vinicius-guedes-santos** [comment, marked as answer]
Ciente

**Reply by PedroMiguel7** [reply]
beleza
